README.md

```diff
@@ -141,50 +142,8 @@ As can be seen in the above output, each event line shows the following informat
 
 Note about string arguments and userspace pointers: as pointers are being dereferenced from userspace memory, a malicious program may change the content being read before it actually gets executed in the kernel. Take this into account when doing security related stuff.
 
-Tracee currently supports the following system calls:
-
-* execve
-* execveat
-* mmap
-* mprotect
-* clone
-* fork
-* vfork
-* newstat
-* newfstat
-* newlstat
-* mknod
-* mknodat
-* dup
-* dup2
-* dup3
-* memfd_create
-* socket
-* close
-* ioctl
-* access
-* faccessat
-* kill
-* listen
-* connect
-* accept
-* accept4
-* bind
-* getsockname
-* prctl
-* ptrace
-* process_vm_writev
-* process_vm_readv
-* init_module
-* finit_module
-* delete_module
-* symlink
-* symlinkat
-* getdents
-* getdents64
-* creat
-* open
-* openat
+Tracee currently supports a subset of system calls events, which can be listed with:<br>
+`./tracee.py -l`
 
 Other supported events are (functions called in kernel space):
```

Comment:
Other events are also listed, so could remove this list and modify line 144 accordingly

Reply:
I kept this because we elaborate a bit more about these events, as these are internal kernel functions, thus less known to user. I don't mind removing it

Reply:
I don't mind either! We can leave it there